api/docs/developer-system-requirements.md

```diff
@@ -14,7 +14,7 @@ We currently only support MacOS and Linux for development and testing.
   * [Ruby](https://www.ruby-lang.org/en/downloads/)
     * Our team's dev/ops scripts are written in Ruby. Most common operations are launched via the project.rb script at the root of each sub-project.
     * You may need to add ruby gems to your path if things are not working. If so follow [these instructions](https://guides.rubygems.org/faqs/#i-installed-gems-with---user-install-and-their-commands-are-not-available).
-  * [Python](https://www.python.org/downloads/) >= 2.7.9
+  * [Python](https://www.python.org/downloads/) >= 3.5
```

Comment:
"Supported versions are Python 3 (3.5 to 3.9)"
https://cloud.google.com/sdk/docs/install

Reply:
We have a few other python files but they all appear to be python 3 👍